help/en/package/jmri/jmrit/simpleclock/SimpleClockFrame.shtml

```diff
@@ -17,27 +17,32 @@
   <link rel="stylesheet" type="text/css" href="/css/print.css"
   media="print">
   <link rel="icon" href="/images/jmri.ico" type="image/png">
-  <link rel="home" title="Home" href="/"><!-- /Style -->
+  <link rel="home" title="Home" href="/">
+  <!-- /Style -->
 </head>
 
 <body>
   <!--#include virtual="/Header" -->
 
   <div class="nomenu" id="mBody">
     <div id="mainContent">
-      <h1>Setup Fast Clock</h1>
+      <h1>Fast Clock Set-up </h1>
 
-      <h2>Introduction to JMRI Fast Clocks</h2>JMRI provides a fast
+      <h2>Introduction to JMRI Fast Clocks</h2>
+      <p>JMRI provides a fast
       clock that can get its time from an internal fast clock or a
       selected hardware fast clock if one exists on your DCC
-      system. This dialog (reached from <b>Setup Fast Clock</b> in
+      system. This dialog (reached from <b>Set up Fast Clock</b> in
       the <b>Clocks</b> submenu under the <b>Tools</b> menu)
       provides for setting options for this fast clock system.
       Information you set up here is saved when you save your
       panels and configuration. This dialog allows you to set start
       up options that take effect when you load your panels and
       configuration. JMRI also has clock displays that can run in
-      stand-alone dialogs or on a PanelPro panel.
+      stand-alone dialogs or on a PanelPro panel.</p>
+      <a href="images/FastClockSetup.png"><img
+      src="images/FastClockSetup.png" width="251" height="327"
+      alt="Fast Clock Setup config pane" align="right"></a>
```

Comment:
here you used setup.

Reply:
See https://github.com/JMRI/JMRI/pull/6188/commits/24a1d33389ebb2ec67e25dfddaea8aea0b3edb9a